packages/pluggable-widgets-tools/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Added
+
+-   We added support for expression returnType to refer to another property and generate its types
```

Comment:
I would mention `assignableTo` specifically, mention it as a feature added in Studio Pro 9.20 and refer to the (future) documentation link. (You may be able to lift some grammatically smooth language from there. 😅)

Reply:
Changes made.

Can it be further improved ?

Reply:
See my other thread :)